docs/technical_reference/unit_models/electrodialysis_0D.rst

```diff
@@ -0,0 +1,219 @@
+Electrodialysis (0D)
+====================
+
+Introduction
+------------
+
+Electrodialysis, an electrochemical separation technology, has been utilized to desalinate water for decades.
+Compared with other technologies, such as reverse osmosis (RO),
+electrodialysis shows advantages in desalinating brackish waters with
+moderate salinity by its less intense energy consumption, higher water recovery, and robust
+tolerance for adverse non-ionic components (e.g., silica and biological substances) in water.
+Between the electrodes of an electrodialysis stack (a reactor module), multiple anion- and
+cation-exchange membranes are alternately positioned and separated by spacers to form individual
+cells. When operated, electrodialysis converts electrical current to ion flux and, assisted by
+the opposite ion selectivity of cation- and anion-exchange membranes (cem and aem), moves ion from
+one cell to its adjacent cell in a cell-pair treatment unit (Figure 1). The ion-departing cell is called a **diluate
+channel** and the ion-entering cell a **concentrate channel**. Recovered (desalinated) water is
+collected from diluate channles of all cell pairs while the concentrate product can be disposed of as brine
+or retreated. More overview of the electrodialysis technology can be found in the *References*.
+
+.. figure:: ../../_static/unit_models/EDdiagram.png
+    :width: 600
+    :align: center
+
+    Figure 1. Schematic representation of an electrodialysis cell pair
+
+
+One cell pair in an electrodialysis stack can thus be treated as a modeling unit that can multiply to
+larger-scale systems.  The presented electrodialysis model establishes mathematical descriptions of
+ion and water transport in a cell pair and expands to simulate a stack with a specified cell-pair number.
+Modeled mass transfer mechanisms include electrical migration and diffusion of ions and osmosis and electroosmosis
+of water. The following key assumptions are based on.
+
+* The concentrate and diluate channels have identical geometry and fluidic conditions.
```

Comment:
What is meant by identical 'fluidic conditions'?

Reply:
I removed these after a second thought.  They were used in some papers to mean the two flows are driven by the same pressure and no water electroosmosis was considered across each other.   This model considers the latter aspect and assumed the pressure condition in the next bullet.  So it's proper to remove them.